VirtoCommerce.Platform.Web/Scripts/app/assets/blades/asset-upload.tpl.html

```diff
@@ -15,7 +15,8 @@
 
                     <!-- 2. ng-file-select | ng-file-select="options" -->
                     <div class="form-input __file">
-                        <input type="file" multiple uploader="uploader" nv-file-select style="z-index: 0;" tabindex="-1">
+                        <input type="file" ng-if="!blade.isImageMode" multiple uploader="uploader" nv-file-select style="z-index: 0;" tabindex="-1">
+                        <input type="file" ng-if="blade.isImageMode" uploader="uploader" nv-file-select style="z-index: 0;" tabindex="-1" accept="image/*">
```

Comment:
Why not add Accept option for Asset uploader control instead of using special isImageMode? I guess is will be more flexible.